### PR TITLE
Add input to provide a tag for image

### PR DESCRIPTION
**Pull request by ericvoshall**
- Adds an input for consumers to provide their own image tag rather than defaulting to the SHA.
- Alphabetize inputs

### CODE REVIEW

File: upload-to-ecr/action.yml [hunk 1]
```diff
@@ -30,18 +26,25 @@ inputs:
...
     required: true
-    description: KMS Key to sign image with
-  vuln-severity-cutoff:
+    description: Path in Parameter Store for Github Token
+  image-tag:
     required: false
-    default: critical
-    description: Severity cutoff for Anchore Scan
+    description: Tag for image
+    default: ${{ github.sha }}
```
Comment (ericvoshall) [inline upload-to-ecr/action.yml:39]: Turns out this doesn't actually work.

I can either make this input required, or have a step that has an output value of the `image-tag` if it's provided or the SHA if it's not.

Comment (dginther) [reply upload-to-ecr/action.yml:39]: Oh, I see, you want the option to provide a tag and not hard-code it to the SHA. We could tag it both with the SHA and also with another user-provided tag, and only perform that step if it was provided?

Comment (ericvoshall) [reply upload-to-ecr/action.yml:39]: I like that approach - I will update this.

Comment (ericvoshall) [reply upload-to-ecr/action.yml:39]: Will the additional tag need to be used for the signing / SBOM steps, or can those continue to just use the SHA?

Comment (dginther) [reply upload-to-ecr/action.yml:39]: I _believe_ that in ECR, the tags are all on the same image, as well as the signature and the SBOM, so it shouldnt be necessary to replicate the steps.

Comment (ericvoshall) [reply upload-to-ecr/action.yml]: Added another step that only runs when the `additional-image-tag` input is provided. Duplicating it doesn't seem ideal, but I wasn't sure of another way to conditionally add the tag.

Confirmed this works (when providing an additional tag as well as omitting it) in one of my team's repos.

**Review by dginther — APPROVED** [review]

**Pull request merged**